releasenotes/notes/0.23/prepare-0.23.0-release-0d954c91143cf9a4.yaml

```
@@ -0,0 +1,124 @@
+---
+prelude: |
+  Qiskit Terra 0.23.0 is a major feature release that includes
+  a multitude of new features and bugfixes. The highlights for this release
+  are:
+
+    * Support for importing `OpenQASM3` programs and creating :class:`.QuantumCircuit` objects
```

What formatting were you aiming at for "OpenQASM 3"?  (I also put a space between "OpenQASM" and "3" like its docs https://openqasm.com/, but I don't think we've got a consistent style in Qiskit.)

---

To be honest it wasn't a conscious decision (or if it was I can't figure out what I was thinking. I'll defer to what's used in the docs:

```suggestion
    * Support for importing OpenQASM 3 programs and creating :class:`.QuantumCircuit` objects
```